diff --git a/node_modules/nuxt3/dist/chunks/index.mjs b/node_modules/nuxt3/dist/chunks/index.mjs
index 9e29b94..b5d6d2f 100644
--- a/node_modules/nuxt3/dist/chunks/index.mjs
+++ b/node_modules/nuxt3/dist/chunks/index.mjs
@@ -1211,6 +1211,15 @@ const appViewTemplate = {

 <body {{ BODY_ATTRS }}>
   {{ APP }}
+  <div
+    aria-live="assertive"
+    class="fixed inset-0 flex items-end px-4 py-6 pointer-events-none sm:p-6 sm:items-start"
+  >
+    <div
+      id="toast-teleport-target"
+      class="w-full flex flex-col items-center space-y-4"
+    ></div>
+  </div>
 </body>

 </html>
