diff --git a/node_modules/toucan-js/dist/index.cjs.js b/node_modules/toucan-js/dist/index.cjs.js
index 0108e0a..8838b44 100644
--- a/node_modules/toucan-js/dist/index.cjs.js
+++ b/node_modules/toucan-js/dist/index.cjs.js
@@ -2,7 +2,6 @@
 
 var core = require('@sentry/core');
 var utils = require('@sentry/utils');
-var uuid = require('uuid');
 var cookie = require('cookie');
 var stacktraceJs = require('stacktrace-js');
 var hub = require('@sentry/hub');
@@ -269,9 +268,9 @@ class Toucan {
      * @param exception An exception-like object.
      * @returns The generated eventId, or undefined if event wasn't scheduled.
      */
-    captureException(exception) {
+    captureException(exception, context = {}) {
         this.debug(() => this.log(`calling captureException`));
-        const event = this.buildEvent({});
+        const event = this.buildEvent(context);
         if (!event)
             return;
         // This is to maintain backwards compatibility for 'event' option. When we remove it, all this complex logic can go away.
@@ -415,7 +414,7 @@ class Toucan {
         const scope = this.getScope();
         // per https://docs.sentry.io/development/sdk-dev/event-payloads/#required-attributes
         const payload = {
-            event_id: uuid.v4().replace(/-/g, ''),
+            event_id: self.crypto.randomUUID().replace(/-/g, ''),
             logger: 'EdgeWorker',
             platform: 'node',
             release,
